src/main/java/uk/gov/hmcts/reform/ccd/documentam/configuration/AuthorisedServicesConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.ccd.documentam.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import uk.gov.hmcts.reform.ccd.documentam.model.AuthorisedServices;
+
+import java.io.InputStream;
+
```

Comment:
you can delete this class now as AuthorisedServices has required config annotations